exemplar-crd/.gitignore

```diff
@@ -0,0 +1,115 @@
+
```

Comment:
do we need this? We're not generating any files that should be ignored

Reply:
IntelliJ artifacts.